package/src/values/__tests__/selector.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import type { AnimatedProp } from "../../renderer/components";
+import { Fill } from "../../renderer/components";
+import { mountCanvas } from "../../renderer/__tests__/setup";
+import { processResult } from "../../__tests__/setup";
+import { Selector } from "../selector";
+
+interface ArrayProps {
+  color: AnimatedProp<string>;
+}
+
+const TestSelector = ({ color }: ArrayProps) => {
```

Comment:
not important but this doesn't need to be wrapped into a component, we can use Fill directly

Reply:
Make sense :) Fixed it.